index.html

```diff
@@ -18,22 +18,125 @@
+        </ul>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio.png" alt="Snapshoot-1" />
+        <p class="project-title">Tonic</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio (1).png" alt="Snapshoot-1" />
+        <p class="project-title">Multi-Post Stories</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio (2).png" alt="Snapshoot-1" />
+        <p class="project-title">Tonic</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio (3).png" alt="Snapshoot-1" />
+        <p class="project-title">Multi-Post Stories</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
```

Comment:
- [ ] I think it would be better if instead use separate `<section>` tags for each card, you use one `<section>` tag to wrap all the cards on your 'work section'. It is not only correct semantically correct but also it is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) :+1: 

- [ ] While you did a wonderful job styling your cards :100: :+1: , it is a project requirement to use 'Grid' to lay out your elements :+1: . [Here](https://css-tricks.com/snippets/css/complete-guide-grid/) you can find more information about grid on CSS.

- [ ] Good job styling all the section works :1st_place_medal: but to match to Figma template chosen you should follow also the background of each card to see the difference between each one :+1: 

**Your design**

![Screenshot from 2023-01-14 11-37-28](https://user-images.githubusercontent.com/105117832/212483975-15dcc89f-34d9-4c06-a9a2-9ef7a5356c84.png)

**Figma template**

![Screenshot from 2023-01-14 11-40-59](https://user-images.githubusercontent.com/105117832/212484019-b7d5ac2d-bb3d-4d6a-a70c-75b32c39f844.png)